docs/src/config/File.md

```diff
@@ -34,6 +34,10 @@ use_keyring = true
 # then set this to false to avoid binding errors
 use_mpris = true
 
+# If set to true, the device name is appended to the debus path for MPRIS,
+# this allows multiple instances of `spotifyd` to run at once with MPRIS on.
+unique
```

Comment:
```suggestion
# If set to true, the device name is appended to the dbus path for MPRIS,
# this allows multiple instances of `spotifyd` to run at once with MPRIS on.
unique_mpris = false
```

Reply:
oh that's funny, I actually had exactly this and must've forgotten to hit save